test/SparqlExpressionTest.cpp

```diff
@@ -1109,20 +1109,10 @@ TEST(SparqlExpression, geoSparqlExpressions) {
   auto checkLong = testUnaryExpression<&makeLongitudeExpression>;
   auto checkDist = std::bind_front(testNaryExpression, &makeDistExpression);
 
-  checkLat(idOrLitOrStringVec({"POINT(24.3 26.8)", "NotAPoint", I(12)}),
-           Ids{D(26.8), U, U});
-  checkLong(idOrLitOrStringVec({D(4.2), "POINT(24.3 26.8)", "NotAPoint"}),
-            Ids{U, D(24.3), U});
-  checkDist(D(0.0), IdOrLiteralOrIri{lit("POINT(24.3 26.8)")},
-            IdOrLiteralOrIri{lit("POINT(24.3 26.8)")});
-  checkDist(U, IdOrLiteralOrIri{lit("POINT(24.3 26.8)")},
-            IdOrLiteralOrIri{I(12)});
-  checkDist(U, IdOrLiteralOrIri{I(12)},
-            IdOrLiteralOrIri{lit("POINT(24.3 26.8)")});
-  checkDist(U, IdOrLiteralOrIri{lit("POINT(24.3 26.8)"s)},
-            IdOrLiteralOrIri{lit("NotAPoint")});
-  checkDist(U, IdOrLiteralOrIri{lit("NotAPoint")},
-            IdOrLiteralOrIri{lit("POINT(24.3 26.8)")});
+  auto v = ValueId::makeFromGeoPoint(GeoPoint(26.8, 24.3));
```

Comment:
```
autot actualLat = Point(fromBits(point.toBits)).lat_;
EXPECT_FLOAT_NEAR(actualLat, 26.8, somePrecision);
// build a DoubleId from the actualLat and pass that to the `checkLat`function as expected
```
Same for long.
add a comment that the toBit and fromBit has some precision loss as there are only 30 bits per coordinate.